docs-building jobs failing

### What happened?

The CI jobs that build the documentation are failing.

### What command did you use?

```shell
n/a
```


### What version of the software are you running?

n/a

### How are you running this software?

Other

### Is your data BIDS valid?

No

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell
Traceback (most recent call last):
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/cmd/build.py", line 298, in build_main
    app.build(args.force_all, args.filenames)
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/application.py", line 355, in build
    self.builder.build_update()
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/builders/__init__.py", line 293, in build_update
    self.build(to_build,
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/builders/__init__.py", line 312, in build
    with logging.pending_warnings():
  File "/home/circleci/.pyenv/versions/3.10.9/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/util/logging.py", line 222, in pending_warnings
    memhandler.flushTo(logger)
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/util/logging.py", line 187, in flushTo
    logger.handle(record)
  File "/home/circleci/.pyenv/versions/3.10.9/lib/python3.10/logging/__init__.py", line 1634, in handle
    self.callHandlers(record)
  File "/home/circleci/.pyenv/versions/3.10.9/lib/python3.10/logging/__init__.py", line 1696, in callHandlers
    hdlr.handle(record)
  File "/home/circleci/.pyenv/versions/3.10.9/lib/python3.10/logging/__init__.py", line 964, in handle
    rv = self.filter(record)
  File "/home/circleci/.pyenv/versions/3.10.9/lib/python3.10/logging/__init__.py", line 821, in filter
    result = f.filter(record)
  File "/tmp/venv/lib/python3.10/site-packages/sphinx/util/logging.py", line 427, in filter
    raise exc
sphinx.errors.SphinxWarning: autodoc: failed to import module 'bspline' from module 'sdcflows.interfaces'; the following exception was raised:
numpy._core.multiarray failed to import

Warning, treated as error:
autodoc: failed to import module 'bspline' from module 'sdcflows.interfaces'; the following exception was raised:
numpy._core.multiarray failed to import
make: *** [Makefile:61: html] Error 2
make: Leaving directory '/tmp/gh-pages/docs'

Exited with code exit status 2
```


### Additional information / screenshots

_No response_

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -48,7 +48,6 @@ autodoc_mock_imports = [
     "nilearn",
     "nipy",
     "nitime",
-    "numpy",
     "pandas",
     "seaborn",
     "skimage",

